### ISSUE TITLE
Train loss issue

### ISSUE THREAD

**Issue by SunQiu88** [EricZgw/PyramidBox#6]
fune train  ,after 200000+ the loss is very large?

**Comment by samehraban** [comment]
@zh199110 I know it's irrelevant but can you help me with #7?

**Comment by EricZgw** [comment]
@zh199110 
You can try to make [learning_rate_decay_factor](https://github.com/EricZgw/PyramidBox/blob/03f67d8422f2f03e5ee580e2f92d4040c24bbdb4/train_model.py#L61) smaller or  change [learning_rate_decay_type](https://github.com/EricZgw/PyramidBox/blob/03f67d8422f2f03e5ee580e2f92d4040c24bbdb4/train_model.py#L401) to 'fixed' and adjust learning rate every few epoches.
